src/main/java/ch/njol/skript/util/VisualEffect.java

```diff
@@ -184,7 +184,16 @@ else if (raw instanceof ItemType) {
 		
 		// 1.11 particles
 		TOTEM("TOTEM"),
-		SPIT("SPIT");
+		SPIT("SPIT"),
+		
+		// 1.13 particles
+		SQUID_INK("SQUID INK"),
+		BUBBLE_POP("BUBBLE POP"),

```

Comment:
Shouldn't these be `SQUID_INK("SQUID_INK")` and `BUBBLE_POP("BUBBLE_POP")`?

Reply:
Yes.

Reply:
Ah yes my bad, this has been fixed!